lib/groups/help.js

```diff
@@ -5,7 +5,7 @@ const commandLineUsage = require('command-line-usage');
 class HelpGroup {
     outputHelp(command = null) {
         if (command !== null) {
-            const commands = require('../utils/commands').info;
+            const { commands } = require('../utils/commands');
```

Comment:
this line changes the meaning.
should not it be `const { info } = require('../utils/commands');`

Reply:
Changed the named export to `module.exports.commands = commands;`

Reply:
Let's go with `commands`. `info` doesn't help at all as name. Good change, thank you!